From: Martin Klapetek <mklapetek@kde.org>
Date: Wed, 24 Feb 2016 01:47:49 +0000
Subject: Bump libaccounts dep versions
X-Git-Tag: v16.03.80
X-Git-Url: http://quickgit.kde.org/?p=kaccounts-integration.git&a=commitdiff&h=a60c1636b6bf0bf5b0718d8de06facfa959ad719
---
Bump libaccounts dep versions

KAccounts now also requires libaccounts-glib 1.21 to ensure the multi-
desktop solution for bug 347219 actually works (which it does only with
>= 1.21)

CCBUG: 347219
---


--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,13 +15,15 @@
 set(KACCOUNTS_SOVERSION "1")
 set(REQUIRED_QT_VERSION "5.2.0")
 set(REQUIRED_KF5_VERSION "5.4.0")
-set(ACCOUNTSQT_DEP_VERSION "1.10")
+set(ACCOUNTSQT_DEP_VERSION "1.13")
 set(SIGNONQT_DEP_VERSION "8.55")
+set(ACCOUNTSGLIB_DEP_VERSION "1.21")
 
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Core Widgets)
 find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED KCMUtils I18n CoreAddons DBusAddons Declarative)
 find_package(AccountsQt5 ${ACCOUNTSQT_DEP_VERSION} REQUIRED CONFIG)
 find_package(SignOnQt5 ${SIGNONQT_DEP_VERSION} REQUIRED CONFIG)
+find_package(LibAccountsGlib ${ACCOUNTSGLIB_DEP_VERSION} REQUIRED)
 
 add_definitions (-fexceptions
                  -DQT_NO_KEYWORDS)
